cmd/proxy/actions/app.go

```diff
@@ -129,14 +131,22 @@ func App() (*buffalo.App, error) {
 		app.Stop(err)
 	}
 	app.Use(T.Middleware())
+
 	if !env.FilterOff() {
 		mf := module.NewFilter()
-		app.Use(newFilterMiddleware(mf))
+		app.Use(mw.NewFilterMiddleware(mf))
+	}
+
+	// Having the hook set means we want to use it
+	if _, ok := env.ValidatorHook(); ok {
```

Comment:
can we inject value into the middleware?
i don't think we will change the env value after the container is up, rather than restarting containers with proper configuration